resources/views/admin/pireps/scripts.blade.php

```diff
@@ -70,12 +70,12 @@
           let new_badge;
           let new_badge_text;
 
-          if (values.new_status === '2') {
+          if (values.new_status === '{{ App\Models\Enums\PirepState::ACCEPTED }}') {
```

Comment:
You don't need the quoting, it should just be a plain int value

Reply:
Yeah tried it but didn't work..